js/app.js

```diff
@@ -1,5 +1,19 @@
 /* globals Utils */
 const STATE = {
+  domains: {
+    domain1: {
```

Comment:
The power of objects is to have the id as key to proper search for an item, for example this could be something like:

```
const domains = {
  "www.reddit.co.uk": {
    domain: "www.reddit.co.uk",
    tiles: ["tile1"],
  },
  "medium.com": {
    domain: "medium.com",
    tiles: ["tile2"],
  },
  "www.youtube.com": {
    domain: "www.youtube.com",
    tiles: ["tile3"],
  },
};
```